src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+  border: 1px solid#F3F3F3;
```

Comment:
In CSS, hexadecimal color codes should not contain uppercase letters. Please use lowercase for consistency as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).